ocs_ci/framework/conf/ocp_version/ocp-4.20-config.yaml

```diff
@@ -13,10 +13,10 @@ ENV_DATA:
   acm_hub_channel: "release-2.15"
   acm_version: "2.15"
   mce_version: "2.10"
-  submariner_version: "0.21.0"
+  submariner_version: "0.22.0"
   # Below values needs to be changed when acm and submariner are GAed
   submariner_source: "downstream"
-  submariner_release_type: "unreleased"
-  acm_hub_unreleased: true
+  submariner_release_type: "released"
+  acm_hub_unreleased: false
```

Comment:
Usually we remove these values from the config after GA
See 
 - [ocp-4.19-config.yaml](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocp_version/ocp-4.19-config.yaml)

 - [ocp-4.19-ga-config.yaml](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocp_version/ocp-4.19-ga-config.yaml)

Reply:
Okay will update